packages/babel-preset-latest/src/index.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function preset(context, opts = {}) {
+  return {
+    presets: [
+      opts.es2015 !== false && [require("babel-preset-es2015"), opts.es2015],
```

Comment:
yeah thats a nice idea, then it should be also easy for people to use this preset env aware kindof by only enabling 2016/2017 for node 6 e.g.